pghoard/walreceiver.py

```diff
@@ -127,6 +127,14 @@ def start_replication(self):
             self.c.start_replication(start_lsn=lsn, timeline=timeline)
         return timeline
 
+    def stop_replication(self) -> None:
```

Comment:
probably there's no urgency to change this right now, but I think this method should be private (it doesn't make sense to call it from outside).